release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -197,6 +197,15 @@ For more information, see xref:../networking/external_dns_operator/nw-installing
 
 SR-IOV support is now available for xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Pensando DSC cards]. OpenShift SR-IOV is supported, but you must set a static, Virtual Function (VF) media access control (MAC) address using the SR-IOV CNI config file when using SR-IOV.
 
+[id="ocp-4-11-support-for-hosted-control-planes"]
```

Comment:
Technology Preview is capitalized. For example:  https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-dynamic-plugin-technology-preview

Otherwise, LGTM!

Reply:
Thanks Ashley!